scout/adapter/mongo/panel.py

```diff
@@ -195,21 +196,25 @@ def add_gene_panel(self, panel_obj, replace=False):
         LOG.debug("Panel saved")
         return result.inserted_id
 
-    def panel(self, panel_id):
+    def panel(
+        self, panel_id: Union[str, ObjectId], projection: Optional[Dict] = None
+    ) -> Optional[Dict]:
         """Fetch a gene panel by '_id'.
 
         Args:
```

Comment:
Now that you defined the typehints I think it is fine to remove both Args and Returns.